app/grapefruit/app.toml

```diff
@@ -294,6 +295,13 @@ port = 11113
 tx = { packets = 3, bytes = 1024 }
 rx = { packets = 3, bytes = 1024 }
 
+[config.net.sockets.fmc_test]
+kind = "udp"
+owner = {name = "fmc_demo", notification = "socket"}
+port = 11114
```

Comment:
This should be reserved in [`oxidecomputer/oana`](https://github.com/oxidecomputer/oana?tab=readme-ov-file#service-processor-udp-ports)

Reply:
Done! Also added gimlet-inspector.